ddtrace/contrib/pytest/plugin.py

```diff
@@ -83,6 +84,14 @@ def pytest_configure(config):
         Pin(tags=ci_tags, _config=ddtrace.config.pytest).onto(config)
 
 
+def pytest_sessionstart(session):
+    pin = Pin.get_from(session.config)
+    if pin is not None:
+        filters = pin.tracer._filters
+        filters += [TraceCiVisibilityFilter()]
+        pin.tracer.configure(settings={"FILTERS": filters})
```

Comment:
@brettlangdon this still wouldn't solve the issue you mention in https://github.com/DataDog/dd-trace-py/pull/3080#discussion_r773905398. Is there anything that is preserved across forks that we can use to keep this filter?

Reply:
re-reviewed, as long as it is a TraceProcessor added via `tracer.configure` it gets added to `Tracer._filters` and will persist across forks.